cms/static/images/pencil-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" role="img" focusable="false" aria-hidden="true">
+  <path d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25ZM21.41 6.34l-3.75-3.75-2.53 2.54 3.75 3.75 2.53-2.54Z" fill="currentColor"></path>
+</svg>
```

Comment:
I'll try to have a real review in the next day, but a quick question: where does this icon come from?

Reply:
Thanks
This icon from Paragon [icons](https://paragon-openedx.netlify.app/foundations/icons/)